store-example06-config-export/start-server.bat

```diff
@@ -0,0 +1,36 @@
+@REM Copyright © 2013 - 2018 Software AG, Darmstadt, Germany and/or its licensors
+@REM
+@REM SPDX-License-Identifier: Apache-2.0
+@REM
+@REM   Licensed under the Apache License, Version 2.0 (the "License");
+@REM   you may not use this file except in compliance with the License.
+@REM   You may obtain a copy of the License at
+@REM
+@REM       http://www.apache.org/licenses/LICENSE-2.0
+@REM
+@REM   Unless required by applicable law or agreed to in writing, software
+@REM   distributed under the License is distributed on an "AS IS" BASIS,
+@REM   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+@REM   See the License for the specific language governing permissions and
+@REM   limitations under the License.
+
+@echo off
+setlocal
+
+set WD=%~d0%~p0
```

Comment:
This line does not mirror the operation in the corresponding shell script.  Use
```
pushd "%~dp0"
set WD=%CD%
popd
```
instead.

Reply:
The rest of the changes look good.